packages/browser-wallet/src/popup/store/utils.ts

```diff
@@ -60,7 +60,7 @@ export function atomWithChromeStorage<V>(
  * @description
  * Create an atom that automatically syncs with chrome local storage.
  */
-export function atomWithChromeStorage<V>(key: ChromeStorageKey, fallback: V, withLoading = false, withSync = false) {
+export function atomWithChromeStorage<V>(key: ChromeStorageKey, fallback: V, withLoading = false, withSync = true) {
```

Comment:
I guess this is the quick-and-dirty way of doing it, but I guess the parameter could be completely removed instead. Or is it because you are uncertain if we might not want it as the default at some point?

Reply:
Yeah I removed it completely. I thought maybe we wanted to let the atom not have sync. But we can add that option back if we need it, I guess. :shrug: